log4j-core/src/main/java/org/apache/logging/log4j/core/appender/MemoryMappedFileManager.java

```diff
@@ -972,15 +932,10 @@ private void drainWriteTouchTasks() {
     }
 
     private void asyncCreateNextRegion(final Region currentRegion, final long newMappingOffsetInFile) {
```

Comment:
This method is no longer "async". You cannot just rename it to "createNextRegion" because there is already a method with this name. So I think this method should be now inlined at its sole usage site.

Reply:
I Will name it as syncCreateNextRegion for now.